### DISCUSSION TITLE
符号学在nft时尚产品中的应用研究

### DISCUSSION THREAD

**Discussion by aniiqing42** [rebase-network/hello-world#41] [General]
大家好，我在做这方面的学术研究，  有兴趣一起讨论啊

**Comment by luckyyang** [comment]
有什么研究成果吗，可以来做一次分享？

**Reply by aniiqing42** [reply]
got it

**Comment by aniiqing42** [comment]
![QQ浏览器截图20220308232830](https://user-images.githubusercontent.com/84902883/157270630-f6e087e7-fed7-4a8a-952a-58594476c4f8.png)

**Comment by luckyyang** [comment]
看起来很有意思，可以加我微信 ljyxxz 组织一次分享

**Reply by jizhouli** [reply]
求分享 +1
